spec/initializers/yaml_autoloader_spec.rb

```diff
@@ -6,6 +6,7 @@
   let(:missing_model)   { model_directory.join("zzz_model.rb") }
 
   before do
+    skip "Not applicable for zeitwerk" if Rails.application.config.autoloader == :zeitwerk
```

Comment:
It's important to note that with rails 6.1, once zeitwerk is loaded, it freezes the various eagerload and autoload paths, causing this test to fail.  Thankfully, we don't need this monkey patch with zeitwerk but it's important to note this change with zeitwerk with 6.1 as we have plugins that manipulate paths so we need to make sure we do it at the correct time or dup the paths.

See: https://github.com/rails/rails/blob/6-1-stable/activesupport/lib/active_support/dependencies/zeitwerk_integration.rb#L106-L110

Reply:
cc @Fryguy @agrare